package.json

```diff
@@ -99,7 +99,7 @@
   },
   "dependencies": {
     "@dojo/framework": "~3.0.0",
-    "@dojo/webpack-contrib": "~3.0.1",
+    "@dojo/webpack-contrib": "~3.0.3",
```

Comment:
Can you update this dependency to `4.0.0-alpha.1`, the bundle analyser changes have been released in that version?